todo_app/data/trello_items.py

```diff
@@ -0,0 +1,28 @@
+import os
+import requests
+from itertools import chain
+from todo_app.models.Item import Item
+
+
+board_id = os.getenv('BOARD_ID')
+api_key = os.getenv("API_KEY")
+token = os.getenv("TOKEN")
+base_url = f'https://api.trello.com/1'
+base_query_params = {
+    'key': api_key,
+    'token': token,
+    'cards': 'open',
+}
+list_url = f'{base_url}/boards/{board_id}/lists'
+base_query_string = f'?key={api_key}&token={token}&cards=open'
+
+def get_cards():
+  lists = requests.get(f'https://api.trello.com/1/boards/{os.getenv("BOARD_ID")}/lists?key={os.getenv("API_KEY")}&token={os.getenv("TOKEN")}&cards=open').json()
+  nested_cards = [[Item.from_trello_card(card, list) for card in list['cards']] for list in lists]
+  return list(chain.from_iterable(nested_cards))
+
+def add_card(card_name):
+  requests.post(f'https://api.trello.com/1/cards?key={os.getenv("API_KEY")}&token={os.getenv("TOKEN")}&cards=open&name={card_name}&idList={os.getenv("TO_DO_LIST_ID")}')
```

Comment:
Ah yes - this is from a bit of playing around I was doing. For some reason, when running the tests, when these are pulled out as variables it's not getting their values correctly - just returning `None`. It works fine when running the code, and also the tests are fine when the `os.getenv` calls are in the request directly. I'm probably going to merge this now after removing the variables so I can get my next PR up, but if you know why this might be I agree it's nicer to have them pulled out as variables!